src/index.html

```diff
@@ -8,7 +8,224 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <!--Header-->
+      <header>
+        <div>
+          <span>Home</span>
+          <span style="text-align: right;">Download My Full Resume</span>
+
+        </div>
+      </header>
+
+      <body>
+        <div id="top">
+          <h1 style="text-align: center">John Roberts</h1><br>
+          <h2 style="text-transform: uppercase;text-align: center">business expert</h2>
+          <img src="touxiang.jpg" alt="picture" style="text-align:center">
+        </div>
+
+        <div>
+          <div id="Slider">
+            <!--Slider About me-->
+            <div id="AboutMe_Container">
+              <div id="AboutMe_title">
```

Comment:
Can it follow the same name convention for the ID names? For example, using `kebab-case` or `snake_case` consistently? But don't mix use upper cases and lower cases for the naming.

Read more about Case Styles: https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841